ui/src/dashboards/components/VEO.tsx

```diff
@@ -58,6 +60,15 @@ interface OwnProps extends WithRouterProps {
 type Props = OwnProps & StateProps & DispatchProps
 
 class VEO extends PureComponent<Props, {}> {
+  public componentDidMount() {
+    const {existingView} = this.props
+    const view = existingView || createView<XYView>(ViewType.XY)
+
+    this.props.setActiveTimeMachine(VEO_TIME_MACHINE_ID, {view})
```

Comment:
is it possible to destructure these on line 64?

Reply:
because the name of the action creator is the same as the name that the prop function was named, destructuring would create a shadowed name.